osu.Game/Scoring/ScoreInfo.cs

```diff
@@ -252,11 +252,12 @@ public IEnumerable<HitResultDisplayStatistic> GetStatisticsForDisplay()
         }
 
         [Serializable]
-        protected class DeserializedMod : IMod
+        protected class DeserializedMod : IMod, IEquatable<DeserializedMod>
         {
             public string Acronym { get; set; }
 
-            public bool Equals(IMod other) => Acronym == other?.Acronym;
+            bool IEquatable<IMod>.Equals(IMod other) => other is DeserializedMod them && Equals(them);
```

Comment:
before I go pushing in changes, is there a reason this is the only one of three classes that got an explicit `IEquatable<IMod>` implementation instead of a normal public one?

Reply:
nope. *I swear to god I had it public.*